static/styles/compose.css

```diff
@@ -613,7 +622,7 @@ input.recipient_box {
     }
 
     .divider {
-        color: hsl(0, 0%, 88%);
+        color: hsl(0, 0%, 75%);
```

Comment:
Darker because the composebox itself is now darker. Could also change as we finalize composebox colors.